internal/factsengine/gatherers/fstab.go

```diff
@@ -0,0 +1,100 @@
+package gatherers
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/d-tux/go-fstab"
+	log "github.com/sirupsen/logrus"
+	"github.com/trento-project/agent/pkg/factsengine/entities"
+)
+
+const (
+	FstabGathererName = "fstab"
+	FstabFilePath     = "/etc/fstab"
+)
+
+// nolint:gochecknoglobals
+var (
+	FstabFileError = entities.FactGatheringError{
+		Type:    "fstab-file-error",
+		Message: "error reading /etc/fstab file",
+	}
+
+	FstabFileDecodingError = entities.FactGatheringError{
+		Type:    "fstab-decoding-error",
+		Message: "error decoding fstab file",
+	}
+)
+
+type FstabEntry struct {
+	Device     string   `json:"device"`
+	MountPoint string   `json:"mount_point"`
+	FS         string   `json:"fs"`
```

Comment:
I would rename this to `FileSystemType` or `Type` (`file_system_type/type`).
It makes it more obvious

Reply:
done